uint/Cargo.toml

```diff
@@ -10,6 +10,9 @@ readme = "README.md"
 edition = "2021"
 rust-version = "1.56.1"
 
+[package.metadata.cross.target.mips64-unknown-linux-muslabi64]
+build-std = true
```

Comment:
Do we really need to add this to the `Cargo.toml`? We can only set this in the test?